src/infrastructure/RestrictionService.js

```diff
@@ -110,6 +131,31 @@ class RestrictionService {
   	};
   }
 
+  /**
+   * Format MosaicAddressRestriction to readable object
+   * @param addressRestrictionDTO
+   * @returns Custom address restriction object
+   */
+  static formatMosaicAddressRestriction = addressRestriction => {
+  	let mosaicAddressRestrictionItem = [];
+
+  	// Convert Map<k,v> to Array
```

Comment:
Seems like `addressRestriction.restrictions` is array already

Reply:
it returned in `Map object`
https://github.com/nemtech/symbol-sdk-typescript-javascript/blob/2012207b735d67fd7954e5f999a6cd79f3427e13/src/model/restriction/MosaicAddressRestriction.ts#L52